install-steps.md

```diff
@@ -512,6 +512,35 @@ sudo cp ./oc /usr/local/bin/oc
 oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
 ~~~
 
+### Create RHCOS images cache (Optional)
+
+The installer needs to download two images, the RHCOS image used by the bootrstrap VM and the RHCOS image used by the installer to provision the different nodes.
+
+When running the installer on a network with limited bandwidth sometimes the installer times out while downloading the images, in order to fix that it is recommended to cache the RHCOS images.
+
+The following steps will install httpd and download the required images:
+
+~~~sh
+# Install and run httpd server
+sudo yum -y install httpd
+sudo systemctl enable --now httpd
+# Get the commit id from the installer it will be used to determine which images do we need to download
+export COMMIT_ID=$(/home/kni/openshift-baremetal-install version | grep '^built from commit' | awk '{print $4}')
+# Get the uri for the RHCOS image that will be deployed on the nodes
+export RHCOS_OPENSTACK_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.openstack.path | sed 's/"//g')
+# Get the uri for the RHCOS image that will be deployed on the bootstrap vm
+export RHCOS_QEMU_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.qemu.path | sed 's/"//g')
+# Get the path where the images are published
+export RHCOS_PATH=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json | jq .baseURI | sed 's/"//g')
+# Get the SHA hash for the RHCOS image that will be deployed on the bootstrap vm
+export RHCOS_QEMU_SHA_UNCOMPRESSED=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq -r '.images.qemu["uncompressed-sha256"]')
+# Get the SHA hash for the RHCOS image that will be deployed on the nodes
+export RHCOS_OPENSTACK_SHA_COMPRESSED=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq -r '.images.openstack.sha256')
+# Download the images and place them on the default apache directory
+sudo curl -L ${RHCOS_PATH}${RHCOS_QEMU_URI} -o /var/www/html/images/$RHCOS_QEMU_URI
```

Comment:
I think either this line or the next is incorrect in terms of the destination directory.  `/var/www/html/images/` vs `/var/www/html/`

Reply:
Yep that was a typo. Fixed now, thanks!